benchmarks/src/CMakeLists.txt

```diff
@@ -1,26 +1,45 @@
-cmake_minimum_required(VERSION 3.10) # Set the minimum required version of CMake
+cmake_minimum_required(VERSION 3.10)
 
-project(SerializationBenchmark) # Name your project
+project(SerializatonTwitterBenchmark) # Name your project
 if (NOT CMAKE_BUILD_TYPE)
   message(STATUS "No build type selected, default to Release")
   set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
 endif()
+
 # Set C++ standard to C++17 or higher if needed
-set(CMAKE_CXX_STANDARD 23)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+#set(CMAKE_CXX_STANDARD 23)
+#set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # Optionally, specify the C++ compiler directly (here we use clang++)
 set(CMAKE_C_COMPILER clang)
 set(CMAKE_CXX_COMPILER clang++)
 
+# set(CMAKE_OSX_ARCHITECTURES "arm64")
+
+# if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+#   set(CMAKE_C_FLAGS_DEBUG "-g -fsanitize=address,undefined -fno-omit-frame-pointer")
+#  set(CMAKE_CXX_FLAGS_DEBUG "-g -fsanitize=address,undefined -fno-omit-frame-pointer")
+# else()
+#   set(CMAKE_C_FLAGS_RELEASE "-O2 -fsanitize=address,undefined")
+#  set(CMAKE_CXX_FLAGS_RELEASE "-O2 -fsanitize=address,undefined")
+#  set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2 -g -fsanitize=address,undefined")
+#  set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g -fsanitize=address,undefined")
+#endif()
+
 # Include directories
 include_directories(${CMAKE_SOURCE_DIR})
 
-# Add executable target
+# Add executable targets
 add_executable(SerializationBenchmark benchmark_serialization.cpp)
+add_executable(SerializationTwitterBenchmark benchmark_serialization_twitter.cpp)
 
 target_compile_options(SerializationBenchmark PRIVATE -freflection -stdlib=libc++ -std=c++26)
-# If you have any libraries to link (e.g., if your project grows and you decide to compile some parts into a library),
-# you would specify them here with target_link_libraries(MyLibraryExample mylib)
+target_compile_options(SerializationTwitterBenchmark PRIVATE -freflection -stdlib=libc++ -std=c++26)
+
+#find_package(CURL REQUIRED)
```

Comment:
```suggestion
```

Using curl was a nice touch, but I think it is simpler to either bundle the files, or, alternatively, you can put the files in a separate repo and use CPM to grab them as needed.